_observing-your-data/ad/dashboards-anomaly-detection.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: default
+title: Integrating anomaly detection with Dashboard
+parent: Anomaly detection
+nav_order: 50
+---
+
+# Integrating anomaly detection with Dashboard
```

Comment:
I know it's opening a can of worms, but when I see Dashboard capitalized, I think it's supposed to be OpenSearch Dashboards. Can we say instead something like, Integrating anomaly detection in visualizations? Or Adding anomaly detection to visualizations?

Reply:
Revised.